src/openzaak/components/catalogi/admin/forms.py

```diff
@@ -49,6 +51,16 @@
 )
 
 
+class ReadOnlyWidget(admin.widgets.ManyToManyRawIdWidget):
```

Comment:
Now I'm wondering how this works for other raw_id fields, doesn't django already have something like this? :thinking:

Reply:
so there is no existing widget in django already that we could reuse here?